src/arrays/creating_arrays/medium/q1/MainTest.java

```diff
@@ -0,0 +1,51 @@
+package arrays.creating_arrays.medium.q1;
+
+import global.BaseTest;
+import global.variables.Clause;
+import global.variables.clauses.*;
+import org.junit.jupiter.api.RepeatedTest;
+import org.junit.jupiter.api.RepetitionInfo;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class MainTest extends BaseTest {
+    // Parsons
+
+    private static final int length = 100;
+    private static final int subtract = 1, add = 0; //swap two numbers for a starting multiplication of 1
+
+    public Clause[] testSentence() {
+        return clauseBuilder();
+    }
+
+    public void runMain() {
+        RandomNum.main(new String[0]);
+    }
+
+    @RepeatedTest(length)
+    void printsCorrectArrayOutput(RepetitionInfo repetitionInfo) {
+        double expected = Double.parseDouble(getItemByName("rand" + (repetitionInfo.getCurrentRepetition() - 1)))
+                * (repetitionInfo.getCurrentRepetition() - subtract);
+        double actual = Double.parseDouble(getItemByName("num" + (repetitionInfo.getCurrentRepetition() - 1)));
+        assertEquals(expected, actual, 0.00001, "Your program does not correctly calculate the final number on row " + repetitionInfo.getCurrentRepetition());
```

Comment:
I think for this, we can change this to be more general, and just say that their program does not correctly multiply the random value by the index.